.github/workflows/ci-analytics.yml

```
@@ -47,3 +47,9 @@ jobs:
 
       - name: Run reports
         run: make sprint-reports
+
+  vulnerability-scans:
+    name: Run API Vulnerability Scans
```

Should it say Analytics?

```suggestion
    name: Run Analytics Vulnerability Scans
```